### PR TITLE
Tc integration

### PR DESCRIPTION
**Pull request by DGuhr**

### CODE REVIEW

File: engine_eval/zanzibar/authz-openfga-ex/test/prbac_mark/mark_export.json [hunk 1]
```diff
@@ -0,0 +1,235 @@
+{
+  "id": "01GK7RA1VW5C1P19H0WEKCY0KB",
```
Comment (DGuhr) [inline engine_eval/zanzibar/authz-openfga-ex/test/prbac_mark/mark_export.json:2]: we could actually delete this, just for comparison: this is an export of the actual openfga playground. didn't want to frickle around with the data model, creating structs and such, so created different files to just use json.unmarshal

File: engine_eval/zanzibar/authz-openfga-ex/go.mod [hunk 2]
```diff
@@ -5,7 +5,155 @@ go 1.19
...
 	github.com/golang/glog v1.0.0
 	github.com/openfga/go-sdk v0.1.1
+	github.com/testcontainers/testcontainers-go v0.16.0
 	gopkg.in/yaml.v2 v2.4.0
+
+)
+
+require (
+	github.com/AlecAivazis/survey/v2 v2.3.6 // indirect
+	github.com/Azure/go-ansiterm v0.0.0-20210617225240-d185dfc1b5a1 // indirect
```
Comment (akoserwal) [inline engine_eval/zanzibar/authz-openfga-ex/go.mod:15]: Not sure, why it require all such imports

**Comment by akoserwal** [comment]
@DGuhr, I suggest you add it as a separate directory (authz-openfga-ex) instead updating the exiting approach?

File: engine_eval/zanzibar/authz-openfga-ex/go.mod (hunk 2 above)
Comment (Rajagopalan-Ranganathan) [reply engine_eval/zanzibar/authz-openfga-ex/go.mod:15]: you can try `go mod tidy` to see if it helps

Comment (DGuhr) [reply engine_eval/zanzibar/authz-openfga-ex/go.mod:15]: nope, already tried, doesn't help. I asked in the testcontainers [slack](https://testcontainers.slack.com/archives/CBWBNR76G/p1671032409726519) about it, let's see what they say about it :)

**Comment by DGuhr** [comment]
@akoserwal done.

**Pull request merged**